src/liballoc/sync.rs

```diff
@@ -1568,6 +1566,13 @@ impl<T> Weak<T> {
     }
 }
 
+// Helper type to allow accessing the reference counts without
+// making any assertions about the data field.
```

Comment:
```suggestion
/// Helper type to allow accessing the reference counts without
/// making any assertions about the data field.
```

Reply:
Does this make a difference for private types?

Reply:
When working in an IDE I assume doc comments are picked up for private types.

You'll notice that many private functions also have doc comments.